Src/libCZI_UnitTests/test_DisplaySettings.cpp

```diff
@@ -154,7 +154,7 @@ TEST(DisplaySettings, WriteDisplaySettingsToDocumentAndReadFromThereAndCompare)
 
     // then, get the XML-string containing the metadata, and put this into the CZI-file
     string xml = metadata_to_be_written->GetXml(true);
-    WriteMetadataInfo writerMdInfo = { 0 };
+    WriteMetadataInfo writerMdInfo;
```

Comment:
I guess zero-initialization was intended here (but was done wrongly, only the first member was zero-initialized).

I'd suggest:

either
`WriteMetadataInfo writerMdInfo{};`

or

```
WriteMetadataInfo writerMdInfo;
writerMdInfo.Clear();
```

Reply:
fixed